lib/sejfguru_web/views/asset_view.ex

```diff
@@ -1,3 +1,22 @@
 defmodule SejfguruWeb.AssetView do
   use SejfguruWeb, :view
+
+  def last_user(asset) do
+    booking = List.last(asset.bookings)
+    if booking do
+      "#{booking.user.first_name} #{booking.user.last_name}"
+    else
+      "N/A"
+    end
+  end
+
+  def last_borrowed_at(asset) do
+    booking = List.last(asset.bookings)
+    if booking do
+      datetime = booking.inserted_at
+      "#{datetime.day}/#{datetime.month}/#{datetime.year} #{datetime.hour}:#{datetime.minute}"
```

Comment:
later it would be good to use here some datetime format function, but I don't think there are such function in the default elixir datetime module

Reply:
Yeah I spent like 10 minutes looking for one. No luck :|